python/cudf/cudf/core/column/string.py

```diff
@@ -3988,6 +3988,25 @@ def _mimic_inplace(self, other_col, inplace=False):
         out = super()._mimic_inplace(other_col, inplace=inplace)
         return out
 
+    def view(self, dtype):
+        if self.null_count > 0:
+            raise ValueError("Can not produce a view of a column with nulls")
```

Comment:
This error should specify string column with nulls.

Reply:
Done 👍